app/FreshRSS.php

```diff
@@ -126,9 +126,7 @@ public static function loadStylesAndScripts() {
 			}
 		}
 		//Use prepend to insert before extensions. Added in reverse order.
-		if (Minz_Request::controllerName() !== 'index') {
-			FreshRSS_View::prependScript(Minz_Url::display('/scripts/extra.js?' . @filemtime(PUBLIC_PATH . '/scripts/extra.js')));
-		}
+		FreshRSS_View::prependScript(Minz_Url::display('/scripts/extra.js?' . @filemtime(PUBLIC_PATH . '/scripts/extra.js')));
```

Comment:
The idea of `extra.js` is to avoid loading some JS in our main view. Can that be avoided?

Reply:
Unfortunately not. Extra.js is needed for the feed configuration.
With https://github.com/FreshRSS/FreshRSS/pull/4226 the configuration slider is not in the main view.

To prevent to start the init_extra() right after the script is loaded I implemented:
```
if (!['normal', 'global', 'reader'].includes(context.current_view)) {
	init_extra();
}
```

Reply:
ping @Alkarex

Reply:
I need to look at it again. I would very much like not to include `extra.js` all the time (that is the main point of having this distinct file)